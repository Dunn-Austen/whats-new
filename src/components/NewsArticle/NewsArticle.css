.news-article {
  border: 3px solid black;
  padding: 10px;
  width: 550px;
}

.news-img {
  height: 350px;
  width: 400px;
}
