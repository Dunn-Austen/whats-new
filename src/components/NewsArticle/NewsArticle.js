import React from 'react';
import './NewsArticle.css';

class NewsArticle extends Component {
  constructor() {
    super();
    this.state = {

    }
  }

  render() {
    return(
      <div className="app">

      </div>
    )
  }
}

export default NewsArticle;
