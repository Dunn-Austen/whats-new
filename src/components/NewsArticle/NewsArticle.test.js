import React from 'react';
import NewsArticle from './NewsArticle';
import { shallow } from 'enzyme';

describe('NewsArticle', () => {

  it('should match the snapshot with all data correctly passed', () => {
    const wrapper = shallow(<NewsArticle
            headline={'A'}
            img={'B'}
            description={'C'}
            url={'D'}
            id={1}
            key={1}
          />);

    expect(wrapper).toMatchSnapshot();
  })
})
