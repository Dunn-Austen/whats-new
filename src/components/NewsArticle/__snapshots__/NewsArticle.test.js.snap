// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewsArticle should match the snapshot with all data correctly passed 1`] = `
<article
  className="news-article"
>
  <h3>
    A
  </h3>
  <img
    alt="A"
    className="news-img"
    src="B"
  />
  <p>
    C
  </p>
  <a
    href="D"
  >
    D
  </a>
</article>
`;
